### ISSUE TITLE
Associating labels implicitly

### ISSUE THREAD

**Issue by LanserH** [w3c/wai-tutorials#719]
Dear reader,

From my work as a researcher, I find that if I do not confirm the advice for developers on page https://www.w3.org/WAI/tutorials/forms/labels/ on the WCAG 2.1 Techniques page (https://www.w3.org/WAI/tutorials/forms/labels/ /www.w3.org/WAI/WCAG21/Techniques/html/H44.html).

This mainly concerns the implicit <label> elements. A customer can say it's built digitally accessible, because W3C.org has that on their tutorial page, but with a comment that explicitly associated labels are better for the helper software. In practice I come across this more and more and I disapprove of the implicit label elements based on the technique page H44.

But if I'm contradicted by a customer because you have an example on the tutorial page, I can't blame a customer. Or can you add on H44's page that the implicit labels are also good?

https://www.w3.org/WAI/WCAG21/Techniques/html/H44.html

**Comment by JAWS-test** [comment]
WCAG techniques (like H44) are always just examples. It is never a violation if a technique is not used. The correct use of a technique only ensures that a SC is fulfilled. But the SC can also be fulfilled in another way. If H44 says that label should be linked to the ID of the form field via the for attribute, that is just one of many options for labeling a form field. Others are:
* aria-label
* aria-labelledby
* enclosing label (https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly)
* title
* ...

**Comment by brianelton** [comment]
Echoing what @JAWS-test noted, the techniques are just examples. Implicit labels are a valid labeling technique.
